components/schema-migrator/migrations/director/20220224153857_add-support-info-to-packages.up.sql

```diff
@@ -0,0 +1,61 @@
+BEGIN;
+
+ALTER TABLE packages
+    ADD COLUMN support_info VARCHAR(256);
```

Comment:
By the spec this is a free text field which can have an arbitrary length, so I think we can remove the length restriction and just leave it as `varchar`